gcovr/configuration.py

```diff
@@ -1081,6 +1081,15 @@ def merge_options_and_set_defaults(
         ),
         action="store_true",
     ),
+    GcovrConfigOption(
+        "json_base",
+        ["--json-base"],
+        group="output_options",
+        metavar="PATH",
+        help="Prepend the given path to all file paths in JSON report.",
```

Comment:
Could also use a mention/example/explanation in the "Merging Coverage Data" user guide page.

Reply:
Will add this.